### ISSUE TITLE
Can't delete galleries with 0.28 and 0.28.1

### ISSUE THREAD

**Issue by Santosus** [Pewpews/happypanda#133]
Hi
When I try to delete any gallery the program freezes

26-02 00:23 INFO   gallery Removing 1 galleries
26-02 00:23 INFO   gallery Attempt to remove: b'(C89)  Sakura-Iro (Fate Stay Night)' by b'Jyouren Kishidan (kiasa)'
26-02 00:23 CRITICAL app      File "c:\python\32-bit\3.4\lib\threading.py", line 889, in _bootstrap
  File "c:\python\32-bit\3.4\lib\threading.py", line 930, in _bootstrap_inner

26-02 00:23 CRITICAL app    <class 'AttributeError'>: 'NoneType' object has no attribute 'write'

everything else seems to be working well.

**Comment by twiddli** [comment]
Hi, I understand that you're running from source. This kind of traceback has always been problematic. Could you please try do this:

In the file `gallery.py` line 1398-ish, remove this part of code:

``` python
                log_i('Attempt to remove: {} by {}'.format(gallery.title.encode(),
                                            gallery.artist.encode()))
```

And then try deleting again. This time, hopefully, a more useful traceback will get printed.

**Comment by Santosus** [comment]
I'm sorry but I don't know how I can do this, I couldn't find this gallery.py file anywhere.

**Comment by twiddli** [comment]
I think I have fixed the issue in the upcoming new version. Please stay tuned~

**Comment by twiddli** [comment]
Please re-open this issue if the hasn't been fixed in v0.29.
